### DISCUSSION TITLE
Custom CSS?

### DISCUSSION THREAD

**Discussion by eemax** [iamcco/markdown-preview.nvim#396] [Q&A] [answered]
Hey guys,

Anyone of you using a custom CSS file that you can share and how you set it up?

I tried a CSS file I found on GitHub but didn't work.

**Comment by julve** [comment]
Try [this](https://github.com/sindresorhus/github-markdown-css/blob/main/github-markdown.css) one

**Reply by eemax** [reply]
Yes that is actually the one I tried. Sorry for not giving more details in my initial post.

I tried that css file and added this to my init.vim:
`let g:mkdp_markdown_css = '~/.config/nvim/github_markdown.css'`

Something is still off though. Maybe it is just the highlighting and I need to use the other setting:
`let g:mkdp_highlight_css = ''`

However, I tried using the css file for both settings and it still seems off. So I was wondering if someone has a working config so I can compare.

**Comment by Vaisakhkm2625** [comment]
Hi,

can anyone help, where can i find all classes to make my own custom css file?

 I really like this obsidian dragula theme [this](https://github.com/dracula/obsidian/blob/main/obsidian.css)

it would be easier if i could get all classes as list or something :)... 

or is there any repo i can find more  css files?

**Comment by cnshsliu** [comment]
Give it a try 
https://github.com/cnshsliu/simple_markdown_preview.nvim
This plugin allows your own CSS file, refer to the README

**Comment by eggbean** [comment, marked as answer]
If anyone is trying to get this working, the solution is to put it in brackets with `expand` before the path if you are using a tilde for home.

    let g:mkdp_markdown_css = expand('~/.config/nvim/css/github-markdown.css')
